var/spack/repos/builtin/packages/iegenlib/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# -----------------------------------------------------------------------------
+# Author: Derick Huth <derick.huth@utah.edu>
+# -----------------------------------------------------------------------------
+#   Note: This package was created by the ctop reaserch group at
+#         the University of Utah, Shool of Computing and is unifiliated
+#         with IEGenLib
+# -----------------------------------------------------------------------------
+from spack import *
+
+
+class Iegenlib(AutotoolsPackage):
+    """Inspector/Executor Generation Library for manipulating sets
+       and relations with uninterpreted function symbols. """
+
+    homepage = "http://github.com/CompOpt4Apps"
+    git      = "https://github.com/CompOpt4Apps/IEGenLib.git"
+
+    version('master', branch='master')
```

Comment:
Do you have any checksummable (non-changing) versions you can add?  That's important for security (see past discussions for why).  If there isn't at least one release of this package, maybe it's not ready for Spack yet...